ui/app/js/app.js

```diff
@@ -433,6 +493,14 @@ angular.module('am').config(
     when('/silences', {
       templateUrl: 'app/partials/silences.html',
       controller: 'SilencesCtrl',
+      resolve: {
+        silences: function(Silence) {
+          // Required to get the total number of silences before the controller
+          // loads. Without this, the user is forced to page 1 of the
+          // pagination.
+          return Silence.query({'n':0});
```

Comment:
dummy request to just get the total number of silences, which is required to allow users to go to a page in the pagination that isn't 1.